services/src/main/java/org/keycloak/services/clientpolicy/condition/ClientRolesCondition.java

```diff
@@ -76,8 +72,14 @@ private boolean isRolesMatched(ClientModel client) {
         return isMatched;
     }
 
-    private List<String> getRolesForMatching() {
-        return componentModel.getConfig().get(TestClientRolesConditionFactory.ROLES);
+    private Stream<String> getRolesStreamForMatching() {
+        if (componentModel.getConfig() == null) return null;
+        List<String> roles = componentModel.getConfig().get(ClientRolesConditionFactory.ROLES);
+        if (roles == null) {
+            return null;
+        } else {
+            return roles.stream();
```

Comment:
There is no need in `else` block here, one can do:
```
 if (roles == null) {
      return null;
}
return roles.stream();
```

Reply:
@valb3r Thank you. I'll revise codes as you pointed out.